### DISCUSSION TITLE
桌面版更换电脑无法登录，换电脑后如何登录桌面版（小白问）

### DISCUSSION THREAD

**Discussion by AlvinLXP** [AlistGo/alist#6239] [Q&A]
电脑更换后，新的电脑一直无法登录；
![image](https://github.com/alist-org/alist/assets/24516032/31f5b6ed-e9c4-406b-95da-ff4377812f50)

**Comment by anwen-anyi** [comment]
你是新电脑 新建的AList 不应该是使用admin管理员账号登录吗
